core/domain/platform_feature_services.py

```diff
@@ -33,23 +33,30 @@
 from __future__ import annotations
 
 from core import feconf
-from core import platform_feature_list
+from core import platform_feature_list as feature_list
 from core.constants import constants
 from core.domain import platform_parameter_domain
+from core.domain import platform_parameter_list as parameter_list
 from core.domain import platform_parameter_registry as registry
 
 from typing import Dict, List, Set
 
-ALL_FEATURES_LIST: List[platform_feature_list.ParamNames] = (
-    platform_feature_list.DEV_FEATURES_LIST +
-    platform_feature_list.TEST_FEATURES_LIST +
-    platform_feature_list.PROD_FEATURES_LIST
+ALL_FEATURES_LIST: List[feature_list.ParamNames] = (
+    feature_list.DEV_FEATURES_LIST +
+    feature_list.TEST_FEATURES_LIST +
+    feature_list.PROD_FEATURES_LIST
 )
 
 ALL_FEATURES_NAMES_SET: Set[str] = set(
     feature.value for feature in ALL_FEATURES_LIST
 )
 
+ALL_PLATFORM_PARAMETERS_EXCEPT_FEATURES: List[parameter_list.ParamNames] = [
+    parameter_list.ParamNames.DUMMY_PARAMETER,
```

Comment:
I think we can get rid of this dummy parameter altogether, is it useful for anything?

Reply:
It would be used in e2e tests.